docs/KlassenTjenestekanal.adoc

```diff
@@ -101,7 +101,7 @@ _The value of this property is the (estimated) time needed for executing a Servi
 | Merknad 1 / _Note 1_ |  Egenskapen brukes bare hvis behandlingstiden er spesifikk for den valgte kanalen.
 
 _This property is used only if the processing time is specific for the channel chosen._ 
-| Merknad 2 / _Note 2_ |  Opplysningen skal oppgis ved hjelp av ISO 8601-syntaksen for varighet. Forklaring er gitt på https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia-siden &#x29C9;, window="_blank", role="ext-link"] som refererer til den offisielle ISO-standarden.
+| Merknad 2 / _Note 2_ |  Opplysningen SKAL oppgis ved hjelp av ISO 8601-syntaksen for varighet. Forklaring er gitt på https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia-siden &#x29C9;, window="_blank", role="ext-link"] som refererer til den offisielle ISO-standarden.
 
 __The actual information is provided using the ISO 8601 syntax for durations. Explanation is provided in the https://en.wikipedia.org/wiki/ISO_8601#Durations[Wikipedia page &#x29C9;, window="_blank", role="ext-link"] that references the official ISO standard.__
```

Comment:
The information MUST be provided using....

Reply:
Kopiert fra EU (EUs måte å si at verdene skal hentes fra ...), men rettes for å være konsekvent med resten av spesifikasjonen vår.